pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -29,6 +29,9 @@ store:
     # Maximum number of open connections to the database
     # `0` value means number of open connections is unlimited
     maxOpenConnections: 0 # ENV: KUMA_STORE_POSTGRES_MAX_OPEN_CONNECTIONS
+    # Maximum number of connections in the idle connection pool
+    # <0 value means no idle connections and 0 means default max idle connections
+    maxIdleConnections: 0  # ENV: KUMA_STORE_POSTGRES_MAX_IDLE_CONNECTIONS   
```

Comment:
Yes, these values are 0 by default and the connections open/idle do not close by its age. If we have idle connections configured, it will use already open connections and will improve performance.